controller/linodemachine_controller.go

```diff
@@ -260,6 +260,16 @@ func (r *LinodeMachineReconciler) reconcileCreate(
 		}
 	}
 
+	// Should we check if the VPC ref in LinodeCluster is ready? Or is it enough to check if the VPC exists?
```

Comment:
Yeah thats what I was thinking. Does the current implementation seem fine? Currently, I'm checking if VPC is ready for LinodeMachine when VPC ref is provided, if it isn't I'm making a check for LinodeCluster VPC ref.

Reply:
I might reverse it to check the cluster first since I imagine that will be the more common use case but otherwise seems fine.

Reply:
Hmmm is that necessary? 

When we do not have a VPC ref in LinodeMachine, we will make a check for LinodeCluster VPC ref's LinodeVPC obj. But, if we have a VPC ref in LinodeMachine, we only care about that LinodeVPC object since that is prioritized.

Reply:
Yeah in the case both are defined it's fine to leave as-is so the machine takes priority